eclair-core/src/main/scala/fr/acinq/eclair/router/NetworkStats.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair.router
+
+import fr.acinq.bitcoin.Satoshi
+import fr.acinq.eclair.wire.ChannelUpdate
+import fr.acinq.eclair.{CltvExpiryDelta, MilliSatoshi}
+
+/**
+ * Created by t-bast on 30/08/2019.
+ */
+
+case class Stats[T](min: T, max: T, median: T, percentile5: T, percentile10: T, percentile25: T, percentile75: T, percentile90: T, percentile95: T)
```

Comment:
There may be off-the-shelf libs for that, e.g. `SummaryStatistics` from apache or guava.

Those could have the added advantage to be incremental, we wouldn't have to use a `TickNetworkStats`.

Reply:
Guava works fine and is more efficient, switching to that!